keyboards/keychron/q0/config.h

```diff
@@ -21,6 +21,11 @@
 /* turn off effects when suspended */
 #define RGB_DISABLE_WHEN_USB_SUSPENDED
 
+/* Encoder Configuration*/
+#ifdef ENCODER_ENABLE
+    #define ENCODER_DEFAULT_POS 0x3
```

Comment:
Is this needed?  IE, does this actually help?

Reply:
If this is asking about the block for the Q0 specifically, I can see now how it's redundant to have it be inside of `keychron/q0/config.h since only one board in the `q0` folder uses it.

If this is asking for why this block exists in the first place and why this is needed, I consider this having less boilerplate code, much like how moving common stuff like RGB effects and the Vendor ID into `[q/v][XX]/info.json` is practical than having them defined in each variants `info.json`. As for the `#define ENCODER_DEFAULT_POS 0x3`, it was for lost ticks that people experienced some time ago, even with it being improved upon with #16932.

Reply:
Moved the `#define ENCODER_DEFAULT_POS 0x3` define over to `q0/rev_0131/config.h`.